pg_search/src/bootstrap/create_bm25.rs

```diff
@@ -465,14 +275,9 @@ unsafe fn delete_bm25_index_by_oid(index_oid: pg_sys::Oid) -> Result<()> {
     Ok(())
 }
 
-#[pg_extern(sql = "
-CREATE OR REPLACE FUNCTION paradedb.index_size_impl(
-    index_name text
-) RETURNS bigint
-LANGUAGE c AS 'MODULE_PATHNAME', '@FUNCTION_NAME@';
-")]
+#[pg_extern]
 fn index_size(index_name: &str) -> Result<i64> {
-    let index_oid = index_oid_from_index_name(&format!("{index_name}_bm25_index"));
+    let index_oid = index_oid_from_index_name(index_name);
```

Comment:
I would prefer this function take a `PgRelation` instead of a string.  The reason is that Postgres will validate the user is providing a valid, and fully-qualified (or SEARCH_PATH included) relation name for us.

```suggestion
    index_name: PgRelation,
```

However, we'll also need to reopen the relation at the start of the function with an AccessShareLock:

```rust
let index_name = PgRelation::with_lock(index_name.oid(), pg_sys::AccessShareLock as _);
```

The reason for reopening it is to ensure a concurrent DROP/ALTER doesn't come in while we're reading it.